vault/logical_system.go

```diff
@@ -2623,6 +2623,11 @@ func (b *SystemBackend) handleConfigUIHeadersUpdate(ctx context.Context, req *lo
 	// Translate the list of values to the valid header string
 	value := http.Header{}
 	for _, v := range values {
+		// ExistCustomResponseHeader support checking for custom headers per listener address
```

Comment:
I think this comment is better suited to a godoc on ExistCustomResponseHeader, that way if we wind up using it elsewhere people will still be able to see it by looking at the definition.

Reply:
Done